src/controllers/__tests__/authenticationController.test.ts

```diff
@@ -695,6 +696,14 @@ describe("AuthenticationController#acs", () => {
       const response = await lollipopActivatedController.acs(validUserPayload);
       response.apply(res);
 
+      expect(mockTelemetryClient.trackEvent).toHaveBeenCalledWith({
+        name: "lollipop.error.acs",
+        properties: expect.objectContaining({
+          assertion_ref: anotherAssertionRef,
+          fiscal_code: sha256(aFiscalCode)
```

Comment:
Can we check the `error` property too?

Reply:
done https://github.com/pagopa/io-backend/pull/996/commits/06eceba5db872dc0e45f3d08e6925e71a721e63e